flyingpigeon/calculation.py

```diff
@@ -0,0 +1,290 @@
+from flyingpigeon.nc_utils import get_values, get_coordinates, get_index_lat, get_variable
```

Comment:
`calculation.py` is not very specific.

Reply:
it included fieldmean, climate change signal calculation, would be the place for robustness, bias, trend substractions etc ... 
may be `values_statistic` is better or `nc_statistic`?